mrjob/spark/mrjob_spark_harness.py

```diff
@@ -140,7 +146,7 @@ def make_job(*args):
             )
 
 
-def _run_step(step, step_num, rdd, make_job):
+def _run_step(step, step_num, rdd, make_job, num_reducers):
```

Comment:
It would be helpful for readability to say `num_reducers=None` so that people can see at a glance that it doesn't have to be a number.